### PR TITLE
Revised tutorial

### PR DESCRIPTION
**Pull request by kathyguerron**
Tutorial review

### CODE REVIEW

**Comment by mcelicalderon** [comment]
LGTM

**Comment by nebtrx** [comment]
LGTM :+1:

File: tutorial/index.jade [hunk 1]
```diff
@@ -84,7 +84,7 @@
     :markdown
       ## Sending the Message
 
-      Here is the method that actually sends the sms. If we want to improve
+      Here is how we actually send the sms. If we want to improve
```
Comment (acamino) [inline tutorial/index.jade]: @kathyguerron _sms_ is an acronym for _Short Message Service_. Could you please capitalise it?

_As a side note you should always capitalise acronyms._

Comment (nebtrx) [inline tutorial/index.jade]: @acamino :+1:

**Comment by acamino** [comment]
Other than the comment I left LGTM.

**Comment by acamino** [comment]
Please don't forget to squash the commits before merging.

**Pull request merged**